init.c

```diff
@@ -9,7 +9,7 @@ void createInitDirStructure(char* root_directory){
     char* pathToYads_config = createFolderIfItDoesntExist("yads_config", root_directory);
 
     if (pathToYads_config == NULL){
-        printf("Error!!! creating yads_config folder\n");
+        printf("Warning: Creating yads_config folder.\n");

```

Comment:
This is supposed to be an error, not a warning, and the program exits (with an error message) from line 13 if we find that the folder  "yads_config" still doesn't exist even when we tried to create it, (line 9) because of some unknown reasons. 

Also, this is the main branch, here we push the final code from the [dev](https://github.com/AryanGitHub/reimagined-yads/tree/dev) and other "feature" [branches](https://github.com/AryanGitHub/reimagined-yads/branches) (the branches which start with the feature keyword), I am very sorry for this confusion. I have not provided the necessary ReadME which states this.
Would love to get Issues and PRs for those branches too.